packages/components/src/range-control/stories/index.js

```diff
@@ -1,145 +1,127 @@
-/**
- * External dependencies
- */
-import styled from '@emotion/styled';
-import { boolean, number, text } from '@storybook/addon-knobs';
-
 /**
  * WordPress dependencies
  */
 import { useState } from '@wordpress/element';
-import { wordpress } from '@wordpress/icons';
+import { styles, wordpress } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import RangeControl from '../index';
-import { COLORS } from '../../utils';
+
+const ICONS = { styles, wordpress };
 
 export default {
 	title: 'Components/RangeControl',
 	component: RangeControl,
+	argTypes: {
+		afterIcon: {
+			control: { type: 'select' },
+			options: Object.keys( ICONS ),
+			mapping: ICONS,
+		},
+		allowReset: { control: { type: 'boolean' } },
+		beforeIcon: {
+			control: { type: 'select' },
+			options: Object.keys( ICONS ),
+			mapping: ICONS,
+		},
+		color: { control: { type: 'color' } },
+		disabled: { control: { type: 'boolean' } },
+		help: { control: { type: 'text' } },
+		marks: { control: { type: 'object' } },
+		min: { control: { type: 'number' } },
+		max: { control: { type: 'number' } },
+		railColor: { control: { type: 'color' } },
+		showTooltip: { control: { type: 'boolean' } },
+		step: { control: { type: 'number' } },
+		trackColor: { control: { type: 'color' } },
+		withInputField: { control: { type: 'boolean' } },
```

Comment:
As always, most of these lines will be unnecessary when the TypeScript conversion is done.